indexer/categories_holder.cpp

```diff
@@ -1,9 +1,7 @@
-#include "categories_holder.hpp"
-
-#include "search_delimiters.hpp"
-#include "search_string_utils.hpp"
-
+#include "indexer/categories_holder.hpp"
 #include "indexer/classificator.hpp"
+#include "indexer/search_delimiters.hpp"
+#include "indexer/search_string_utils.hpp"
```

Comment:
OK, we decided to temporary move search_delimiters and search_string_utils to indexer.